skrifa/src/bitmap.rs

```diff
@@ -0,0 +1,485 @@
+//! Bitmap strikes and glyphs.
+
+use super::{instance::Size, metrics::GlyphMetrics, MetadataProvider};
+use crate::prelude::LocationRef;
+use raw::{
+    tables::{bitmap, cbdt, cblc, ebdt, eblc, sbix},
+    types::{GlyphId, Tag},
+    FontData, TableProvider,
+};
+
+/// Set of strikes, each containing embedded bitmaps of a single size.
+#[derive(Clone)]
+pub struct BitmapStrikes<'a>(StrikesKind<'a>);
+
+impl<'a> BitmapStrikes<'a> {
+    /// Creates a new `BitmapStrikes` for the given font.
+    ///
+    /// This will prefer `sbix`, `CBDT`, and `CBLC` formats in that order.
+    ///
+    /// To select a specific format, use [`with_format`](Self::with_format).
+    pub fn new(font: &(impl TableProvider<'a> + MetadataProvider<'a>)) -> Self {
```

Comment:
Missed this but let's have this take `&FontRef<'a>` since we've effectively phased out `TableProvider` in skrifa constructors.